cmake/sysbuild/bootconf.cmake

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+function(setup_bootconf_data)
+    add_custom_target(bootconf_target
```

Comment:
cmake indent is 2 spaces, also move file up a folder as this does not run in sysbuild context